### PR TITLE
882 change new group form

### PR DESCRIPTION
**Pull request by LisaJD**
For issue #882 
1. Changed Name to Name of Group 
2. Added help text to Name of group
3. Added text above 'Website'

### CODE REVIEW

File: eahub/base/static/styles/main.css [hunk 1]
```diff
@@ -460,20 +460,28 @@ body {
 }
 
 .key-facts .background {
-  background-color: rgba(81, 43, 82, 0.7);
+  /* background-color: rgba(81, 43, 82, 0.7); */
```
Comment (sebgrebe) [inline eahub/base/static/styles/main.css:463]: remove comment

File: eahub/base/static/styles/main.css [hunk 2]
```diff
@@ -460,20 +460,28 @@ body {
...
+  background-color: #F7F6F6;
 }
 
 .key-facts .list-group-item {
   background-color: inherit;
   border: none;
-  color: white;
+  /*color: white;*/
```
Comment (sebgrebe) [inline eahub/base/static/styles/main.css:470]: remove comment

File: eahub/localgroups/models.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ class LocalGroup(models.Model):
 
     slug = autoslug.AutoSlugField(populate_from="name", unique=True)
     is_public = models.BooleanField(default=True)
-    name = models.CharField(max_length=100)
+    name = models.CharField("name of group", max_length=100, help_text="University groups: Ideally avoid acronyms in your group name unless they are likely to be unique worldwide. If the name of your university is also the name of a city, indicate in the name that this is a university group.")
```
Comment (sebgrebe) [inline eahub/localgroups/models.py:25]: Would it be possible to make this text in italics, non-bold, to distinguish it from the titles?

**Comment by sebgrebe** [comment]
@LisaJD there's a few minor things I'd change (see my comments), but generally this looks good!
Could you take a screenshot of the form and run this by @cafelow as she asked for the change originally? Once she's approved and you've implemented my comments, we can merge

**Comment by LisaJD** [comment]
Hi @cafelow - here's a screenshot of the form, please let me know if you are happy with the changes or if you want any further edits. image

**Comment by cafelow** [comment]
Looks good to me! Thanks Lisa!

**Pull request merged**